docs/cicd/others-script-ci-template.sh

```diff
@@ -0,0 +1,25 @@
+# This is just an example workflow for continuous integration.
```

Comment:
`#!/usr/bin/env bash` and `set -euxo pipefail`

Reply:
updated